.expeditor/end_to_end.pipeline.yml

```diff
@@ -391,7 +391,7 @@ steps:
         docker:
           privileged: true
           environment:
-            - ACCEPTANCE_HAB_AUTH_TOKEN
+            - PIPELINE_HAB_AUTH_TOKEN
             - HAB_BLDR_URL=https://bldr.acceptance.habitat.sh
```

Comment:
There are a few instances still referencing the acceptance environment in this pipeline definition

Reply:
Yikes! :grimacing: 

Will fix!